dts/bindings/led_strip/ti,tlc5971.yaml

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2021 Esco Medical ApS
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Texas Instruments TLC5971 (and compatible) LED controller
+
+  Driver bindings for daisy chains of a TLC5971 devices using a single device
+  tree node. Length of daisy chains in pixels is defined by the chain-length
+  property.
+
+  Individual global brightness control can be set in devicetree and individial
+  pixel brightness can be set using the led_strip api.
+
+  The pixel order of the TLC5971 is GBR.
+
+  Example device tree node w. 24 pixels (6 TLC5971 devices):
+
+    &spi1 {
+      led_controller: led_controller@0 {
+        status = "okay";
+        compatible = "ti,tlc59711", "ti,tlc5971";
+        label = "led controller";
+        spi-max-frequency = <10000000>;
+        reg = <0x0>;
+        chain-length = <24>;
+        color-mapping = <LED_COLOR_ID_BLUE>,
+                        <LED_COLOR_ID_GREEN>,
+                        <LED_COLOR_ID_RED>;
+      };
+    };
+
+compatible: "ti,tlc5971"
+
+include: spi-device.yaml
+
+properties:
+  chain-length:
+    type: int
+    default: 4
+    description: |
+      The number of RGB LED's in the daisy-chain.
+      Must be multiple of 4. A single TLC5971 device supports 4 RGB LED's.
+
+  color-mapping:
+    type: array
+    required: true
+    description: |
+      Channel to color mapping (or pixel order).
+      For example a BGR channel to color mapping would be
+
+         color-mapping = <LED_COLOR_ID_BLUE
+                          LED_COLOR_ID_GREEN
+                          LED_COLOR_ID_RED>;
+
+  global-brightness-control-red:
+    type: int
+    default: 127
```

Comment:
The unjustified use of defaults in this property and the below properties is also a violation of the rules -- please either justify the defaults as described in the above link or adjust the binding as needed.